src/System.Security.Cryptography.Cng/src/System.Security.Cryptography.Cng.csproj

```diff
@@ -261,6 +261,9 @@
     <ProjectReference Include="..\..\System.Runtime.Extensions\pkg\System.Runtime.Extensions.pkgproj" />
     <!-- ToDo: Remove this P2P reference once new packages are produced and updated -->
     <ProjectReference Include="..\..\System.IO\pkg\System.IO.pkgproj" />
```

Comment:
I think they can be removed. I will file an issue to remove all of them at once.